### ISSUE TITLE
Installation in slim Docker container fails - No matching distribution found for mediapipe

### ISSUE THREAD

**Issue by AmitMY** [google/mediapipe#3070]
**System information** (Please provide as much relevant information as possible)
- OS Platform and Distribution (e.g. Linux Ubuntu 16.04, Android 11, iOS 14.4): `Docker`
- Programming Language and version ( e.g. C++ 14, Python 3.6, Java ): `python`
- Installed using virtualenv? pip? Conda? (if python):`pip`, `conda`, `venv` (tried all)
- [MediaPipe version](https://github.com/google/mediapipe/releases): 0.8.9.1

**Describe the problem**:
I would like to run python `mediapipe` in a docker container.
Using the [`google/mediapipe/Dockerfile`](https://github.com/google/mediapipe/blob/master/Dockerfile) to compile from source is overkill, and Google do not publish a docker image, to the best of my knowledge.

The most basic and slim python Dockerfile to install mediapipe would be
```Dockerfile
FROM python:3.10-slim
RUN pip install mediapipe
```

However, I get the following error:
> No matching distribution found for mediapipe

**[Provide the exact sequence of commands / steps that you executed before running into the problem](https://google.github.io/mediapipe/getting_started/getting_started.html):**
```sh
docker build .
```

I have also tried setting up a `venv` in an `ubuntu` container, or a `conda` from the `miniconda` image, but no luck, the error stays.

**Complete Logs:**
 > [2/2] RUN pip install mediapipe:
#9 0.557 ERROR: Could not find a version that satisfies the requirement mediapipe (from versions: none)
#9 0.557 ERROR: No matching distribution found for mediapipe

### tl;dr
How would one install `mediapipe` for python in Docker, minimally?

**Comment by sgowroji** [comment]
Hi @AmitMY, Present we have docker implementation for Desktop and Android examples as mentioned here https://google.github.io/mediapipe/getting_started/install.html#installing-using-docker. We don't have any such for Python yet. We will mark this as feature request and share your query with the team.

**Comment by AmitMY** [comment]
What I'm trying to understand is why would pip not have a matching distribution for mediapipe?
I can install it with pip just fine on a centos server, or on my mac, or on my windows, but not in a docker?

**Comment by AmitMY** [comment]
This seems to be because I am using arm64.
Doing this instead, works:
```sh
docker build --platform linux/arm64 .
```
